lib/graphql/transfer.ts

```diff
@@ -24,7 +24,9 @@ const TRANSFER_FIELDS = `
   purpose
   e2eId
   reoccurrence
-  nextOccurrence
+  nextOccurrence,
```

Comment:
it would work with `,` but for consistency do you think we could remove them? Just line return is enough to separate fields

Reply:
🤦‍♀️ didn't notice it

Reply:
Not a problem 🙂 We have also not been always consistent with this across the repo I think